Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/StructurePanel.java

```diff
@@ -1924,11 +1919,7 @@ public boolean isAssignableSeveralTimes() {
         if (Objects.isNull(selectedLogicalNode)) {
             return false;
         }
-        if (isHideMediaInLogicalTree()) {
-            return isAssignableSeveralTimes(getSelectedView(selectedLogicalNode));
-        } else {
-            return isAssignableSeveralTimes(selectedLogicalNode);
-        }
+        return isAssignableSeveralTimes(selectedLogicalNode);
```

Comment:
This is a recursive call and the only one exit statement is false when the node is null. Is this really correct or missed I something?

Reply:
This is not a recource call. It calls an different method with the same name, but with a parameter (so an overloaded function).